spec/support/share_db_connection.rb

```diff
@@ -0,0 +1,12 @@
+class ActiveRecord::Base
```

Comment:
exclude this file from rubocop

http://rubocop.readthedocs.io/en/latest/configuration/#includingexcluding-files